### PR TITLE
Reset input value onclick (fix webkit caching again)

### PR DESCRIPTION
**Pull request by jrowlingson**
Related: https://github.com/funkensturm/ember-cli-file-picker/pull/132

### CODE REVIEW

File: addon/components/file-picker.js [hunk 1]
```diff
@@ -51,6 +51,12 @@ export default Component.extend({
     this.$('.file-picker__input').on(
       'change', bind(this, 'filesSelected')
     );
+
+    // reset the input's value to trigger the `onchange` event in the case that the
+    // same file path is selected.
+    document.querySelector('.file-picker__input').addEventListener('click', (e) => {
+      e.target.value = '';
+    });
```
Comment (jrowlingson) [inline addon/components/file-picker.js:59]: Omitting jQuery assuming this addon is going to head in that direction.

File: addon/components/file-picker.js [hunk 2]
```diff
@@ -96,10 +102,6 @@ export default Component.extend({
           });
       }
     }
-
-    if (!this.get('hideFileInput')) {
-      this.$('.file-picker__input').val(null);
-    }
```
Comment (jrowlingson) [inline addon/components/file-picker.js:102]: It was unclear to me why the visibility of the input control was taken into account here. It seems that we'll want to reset the input's value regardless of whether the input control is hidden or not.

This was also clearing the value _after_ the selection. So if the control was visible the selected file path would not be shown.

Comment (fsmanuel) [reply addon/components/file-picker.js:102]: @jrowlingson I guess that was a way to fix the webkit caching but you are totally right that it shouldn't depend on the visibility.

File: addon/components/file-picker.js (hunk 1 above)
Comment (fsmanuel) [reply addon/components/file-picker.js:59]: Does a click trigger the `onchange` or does the `onchange` only fire on user input?

**Review by fsmanuel — COMMENTED** [review]
@jrowlingson Thanks for the PR. I added a question but overall looks good and I guess we can merge it after you answered the question.

**Pull request closed without merging**